src/indieweb_utils/feeds/discovery.py

```diff
@@ -47,12 +49,16 @@ def discover_web_page_feeds(url: str, user_mime_types: Optional[List[str]] = Non
         url = "https://" + url
     elif url.startswith("//"):
         url = "https:" + url
-    try:
-        web_page_request = requests.get(url, timeout=10, allow_redirects=True)
 
+    if web_page_request:
         web_page = web_page_request.text
-    except requests.exceptions.RequestException:
-        return []
+    else:
+        try:
+            web_page_request = requests.get(url, timeout=10, allow_redirects=True)
+
+            web_page = web_page_request.text
```

Comment:
Two main thoughts:

1. It's best to wrap _just_ what you're expecting to throw an exception in a `try/except`. In this case, it'd be better to add an `else` block for line 59.
2. Is there a reason why we're catching `RequestException` then simply re-raising it?